### ISSUE TITLE
Brouter/Navigationmenu/Measure Distance Tool Differences in Navigations Hints

### ISSUE THREAD

**Issue by cannothing** [osmandapp/Osmand#7353]
Version 3.4.4.
What I am doing?
I use a navigationprofile which is based on Brouter
1) I create a route with the navigationmenu and save the calculated route as a gpx-file (nav.gpx)
2.) I create the same route with the measure distance tool and save the calculated route as a gpx-file (measure.gpx.)
3) I start navigation and choose nav.gpx. The navigation hints are the same like in the calculated route from point 1. This are the hints which a created from brouter.
4)  I start navigation and choose measure.gpx. Now I have different navigations hint compared to nav.gpx.

But I used in both cases for calculation Brouter. So I think the navigationshints must be the same.

**Comment by vshcherb** [comment]
Not sure... To be honest navigating by GPX produces a route without turn options so it is simply missing I think. It is the same issue for OsmAnd routing.

**Comment by cannothing** [comment]
I can't imagine that the main devolper of osmand do not know a feature that I know and using since seven years in his app.  If I followed a gpx-track since 2012, I got navigation hints. So what is the meaning of: "To be honest navigating by GPX produces a route without turn options so it is simply missing I think."

 

Do you really know your app so badly?

 

Von: vshcherb <notifications@github.com> 
Gesendet: Mittwoch, 31. Juli 2019 17:25
An: osmandapp/Osmand <Osmand@noreply.github.com>
Cc: cannothing <ich@stephan-zitzmann.de>; Author <author@noreply.github.com>
Betreff: Re: [osmandapp/Osmand] Brouter/Navigationmenu/Measure Distance Tool Differences in Navigations Hints (#7353)

 

Not sure... To be honest navigating by GPX produces a route without turn options so it is simply missing I think. It is the same issue for OsmAnd routing.

**Comment by vshcherb** [comment]
Yes but navigation using GPX is not the same as navigation by OSM-data, GPX navigation simply missing OSM-data so it creates tons of small different issues

**Comment by cannothing** [comment]
As so often, you proof, you do not know your own app. If I am using a gpx-route, the offline routing create a route between the few points of the route using OSM-date. Fun fact you startet to develop this feature.

 

Von: vshcherb <notifications@github.com> 
Gesendet: Freitag, 6. September 2019 10:02
An: osmandapp/Osmand <Osmand@noreply.github.com>
Cc: cannothing <ich@stephan-zitzmann.de>; Author <author@noreply.github.com>
Betreff: Re: [osmandapp/Osmand] Brouter/Navigationmenu/Measure Distance Tool Differences in Navigations Hints (#7353)

 

Yes but navigation using GPX is not the same as navigation by OSM-data, GPX navigation simply missing OSM-data so it creates tons of small different issues
